app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str):
+    new_file = command.split(" ")
+    new_file.remove(new_file[0])
```

Comment:
Why remove the command?(

Reply:
![image](https://user-images.githubusercontent.com/110998159/191735292-b20c000c-600e-45b5-b2ba-b5b3ddf7cf60.png)
i deleted "cp" from string

Reply:
You don't need to delete this part of the command
Because of this your code doesn't work :( 
<img width="649" alt="Screenshot 2022-09-22 at 16 04 04" src="https://user-images.githubusercontent.com/44612098/191754450-2e00f708-71b8-4e80-b578-a9277ca98c93.png">